tree/ntuple/v7/src/RNTupleProcessor.cxx

```diff
@@ -171,33 +213,19 @@ void ROOT::Experimental::RNTupleSingleProcessor::Connect()
 
 //------------------------------------------------------------------------------
 
-ROOT::Experimental::RNTupleChainProcessor::RNTupleChainProcessor(const std::vector<RNTupleOpenSpec> &ntuples,
-                                                                 std::unique_ptr<RNTupleModel> model)
-   : RNTupleProcessor(ntuples, std::move(model))
+ROOT::Experimental::RNTupleChainProcessor::RNTupleChainProcessor(
+   std::vector<std::unique_ptr<RNTupleProcessor>> &processors, std::unique_ptr<RNTupleModel> model)
```

Comment:
`processors` should be passed by value since the caller is relinquishing its ownership